charts/internal/shoot-cert-management-seed/templates/configmap-observability.yaml

```diff
@@ -30,7 +30,10 @@ data:
       - source_labels: [ __name__ ]
         regex: ^(cert_management_.+)$
         action: keep
-
+      - source_labels: [ __name__, code ]
+        regex: ^(promhttp_metric_handler_requests_total);(200|500|503)$
+        target_label: __name__
+        replacement: cert_controller_manager_$1
```

Comment:
You also need to adapt https://github.com/gardener/gardener-extension-shoot-cert-service/blob/457244509ddab8c2bd2f7cd71213bf0c56a2bab7/charts/internal/shoot-cert-management-seed/templates/servicemonitor.yaml#L20-L23